mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -452,6 +453,26 @@ func resourceMongoDBAtlasClusterCreate(d *schema.ResourceData, meta interface{})
 		clusterRequest.NumShards = pointy.Int64(cast.ToInt64(n))
 	}
 
+	if providerName == "AWS" {
+		backup, backupOk := d.GetOk("backup_enabled")
+		providerbackup, providerbackupOk := d.GetOk("provider_backup_enabled")
+		if !backupOk && !providerbackupOk {
+			return errors.New("For AWS cluster one of backup_enabled or provider_backup_enabled must be configured")
```

Comment:
Not correct, you do not have to setup backup.  both are optional in the API because a customer does not have to have backup.

Reply:
I think it is just a validation to check if both are set but, since conflicts with backup_enabled and provider_backup_enabled are set, this validation is not needed.

Could you please test what happens if you remove the validation and set both arguments.